scrapers/velvetveronica.yml

```
@@ -0,0 +1,37 @@
+name: velvetveronica
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - velvetveronica.com/product
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@class="product_title entry-title"]/text()
+        postProcess:
+          - replace:
+            - regex: Video \| (.*) (\[|\().*(\]|\))
+              with: $1
+      Date:
+        selector: //meta[@property="og:image"][1]/@content
+        postProcess:
+          - replace:
+            - regex: .*/uploads/([0-9]{4})/([0-9]{2})/.*
+              with: $1-$2-01
+          #- parseDate: "January 2, 2006"
+      Details:
+        selector: //div[@id="x-legacy-panel-1"]/p or //div[@id="x-legacy-panel-1"]//li
+        concat: "\n\n"
+      Performers: 
+        Name:
+          fixed: Veronica Dyer
+      Tags:
+        Name:
+          selector: //span[@class="tagged_as"]/a/text()
+      Studio:
+        Name: 
+          selector: //meta[@property="og:site_name"]/@content
+      Image: //meta[@property="og:image"]/@content
```

```suggestion
      Image: //meta[@name="twitter:image"]/@content
```
This returns a cleaner result for the image, og:image returns 6 or 7 different results.